spec/controllers/state_file/questions/nj_tenant_eligibility_controller_spec.rb

```diff
@@ -35,7 +35,7 @@
     end
 
     context "when a user is MFS" do
-      let(:intake) { create :state_file_nj_intake, :married_filing_separately }
+      let(:intake) { create :state_file_nj_intake, :df_data_mfs }
```

Comment:
The tests failed for some cases without the full df import. I'm a little fuzzy on why, but it's possible we can simplify the intake factory going forward

Reply:
[sand] Just to clarify, by "full DF import" do you mean df json data that matches the df xml data and is fully fleshed out to match the test case? The factory makes a default version of both json and xml, but that does mean it doesn't always match if MFS/MFJ is changed in the test setup. Some tests are ok with that, but some aren't if they need to use more spouse information. 

I think in the future we should aim for one baseline test trait for each filing status to avoid mismatched data between data sources.

Reply:
Made a ticket to dedupe traits